tsconfig.json

```diff
@@ -0,0 +1,29 @@
+{
+  "compilerOptions": {
+    "allowJs": true,
+    "allowSyntheticDefaultImports": true,
+    "esModuleInterop": true,
+    "isolatedModules": true,
+    "jsx": "react-native",
+    "lib": ["es6"],
+    "moduleResolution": "node",
+    "noEmit": true,
+    "strict": true,
+    "target": "esnext",
+    "baseUrl": ".",
+    "paths": {
+      "@rainbow-me/hooks": ["src/hooks"],
+      "@rainbow-me/routes": ["src/navigation/routesNames"],
+      "@rainbow-me/styles": ["src/styles"],
```

Comment:
only thought i have is itd be nice if we could find a clean way to keep these path aliases in sync between this new `tsconfig.json` and our `babel.config.js`